luigi/contrib/sqla.py

```diff
@@ -189,6 +189,9 @@ def __init__(self, connection_string, target_table, update_id, echo=False, conne
         self.connect_args = connect_args
         self.marker_table_bound = None
 
+    def __str__(self):
+        return self.marker_table_bound or self.connection_string
```

Comment:
i don't follow why you prioritize the use of `self.marker_table_bound` here. if it's non-null, it's a sqlalchemy table object referencing the marker table, not the `target_table`

actually, why wouldn't this `__str__` representation not just be `self.target_table`?

Reply:
```suggestion
        return self.target_table
```

You're right, I missed it